cmd/root.go

```diff
@@ -29,8 +29,11 @@ func init() {
 	rootCmd.Flags().Bool("wal", false, "Turn on WAL mode for sqlite")
 
 	// This is for testing purposes
-	rootCmd.PersistentFlags().Bool("testing", false, "If this flag is set, all v2 activations heights are set to 0.")
+	rootCmd.PersistentFlags().Bool("testing", false, "If this flag is set, all activations heights are set to 0.")
 	rootCmd.PersistentFlags().Int("act", -1, "Able to manually set the activation heights")
+
+	// TEMPORARY
+	rootCmd.PersistentFlags().Int("onf", 0, "If this flag is set, OneWayFactoid act height is set to this")
```

Comment:
if it's temporary, should this be removed before the release?

Reply:
Yes, I put it in for Veenas testing. I was going to remove it this morning